persdict.dat

```diff
@@ -62,3 +62,16 @@ sponte
 squacquerone
 subìto
 username
+snippet
+canvas
+WordPress
+popup
+LibreOffice
+evidenziazione
+color
+tablet
+Layout
+mozillians
```

Comment:
Qui ho il dubbio in merito al fatto che si tratti di qualcosa estremamente specifico per il nostro gruppo. Sono più del parere comunque di inserirlo (viene certamente utilizzato, anche se solo da uno specifico tipo di persone).